pkg/server/router.go

```diff
@@ -57,6 +58,11 @@ func (r *Router) Routes(h router.Handlers) http.Handler {
 
 	btHealthyHandler := backuptarget.NewHealthyHandler(r.scaled)
 	m.Path("/v1/harvester/backuptarget/healthz").Methods("GET").Handler(btHealthyHandler)
+
+	ulGenerateHandler := upgradelog.NewGenerateHandler(r.scaled)
+	m.Path("/v1/harvester/upgradelog/{upgradeLogName}").Methods("POST").Handler(ulGenerateHandler)
```

Comment:
Since `UpgradeLog` is a harvester resource, do we want to use `RegisterSchema` for it like other resources? So we can have API requests with `action` in parameters.

```
curl -u "${CATTLE_ACCESS_KEY}:${CATTLE_SECRET_KEY}" --insecure -X POST https://<harvester-vip>/v1/harvester/upgradelog/<upgradelog-name>?action=generateArchive'
```

https://github.com/harvester/harvester/blob/d8b5b546c5db7079e650f034649cbabe1b53fd95/pkg/api/register.go#L31-L39

Reply:
@FrankYang0529 Thanks for the suggestion! I updated the APIs to align with other resources, please take a look.